src/rt.rs

```diff
@@ -8,9 +8,15 @@ use ffi;
 use glib::translate::{from_glib_none};
 use glib::{to_bool, to_gboolean};
 
-pub fn init() {
+/// Call this function before using any other GTK+ functions.
+/// Note that this function calls gtk_init_check() rather than gtk_init(),
```

Comment:
Can you check the rendered docs? I seem to recall there needs to be an empty comment line after the summary...

Also being more clear about what the `Err` variant means might be nice.